PRD.md

```
@@ -26,147 +26,134 @@ Solo developer (initially: the author) who:
+
 - `MAX_FOCUSES = 5`, `MAX_TASKS_PER_FOCUS = 7` (configurable in `settings.yaml`).
-- Writes that exceed caps still succeed (no hard rejection — markdown is canonical and user might edit anyway), but flip an over-cap flag.
-- Widget shows red badge while over.
+- Writes exceeding caps succeed but flip over-cap flag.
+- Tray icon shows red badge while over.
 - macOS notification fires once per `under → over` transition.
 
 ### FR7 — Configuration
+
 `~/.adhd-ranch/settings.yaml`:
 ```yaml
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)